src/docs/contributing/platforms/common_content.mdx

```diff
@@ -0,0 +1,135 @@
+
+Common content lives in the `/platforms/common/` directory for all platforms, or the `<platform_name>/common/` for content specific to a platform and its frameworks (which we call guides or children).
+
+## Hierarchy
+
+What displays for the user relies upon the hierarchy of content as detailed in our content discussing [Platform Portals](/contributing/platforms/). In short, a guide's content displays if provided. If not provided, then the platform content displays. If the platform content is not provided, then the common content for all platforms displays.
+
+## Tuning Common Content
+
+To ensure that common content can scale as we suppport more platforms and frameworks, we use `include` files. These files can range from, most typically, a code sample specific to a platform or framework to information that augments the core content for a specific platform or framework.
+
+### Simple Example
+
+Let's look at the `/platforms/common/configuration/environments.mdx` file as a simple example of a code sample tuning the common content:
+
+    ---
+    title: Environments
+    sidebar_order: 5
+    description: "Learn how to configure your SDK to tell Sentry about your environments."
+    notSupported:
+     - perl
+    ---
+
+    // frontmatter controls the title of the page, the order in which
+       the page appears in the lefthand sidebar, provides a description
+       for the automatically-generated index page, and identifies if
+       this feature isn't supported by a particular framework, platform,
+       or SDK.
+
+    Sentry automatically creates environments when it receives an event with the
+    environment tag. Environments are case sensitive. The environment name
+    can't contain newlines, spaces or forward slashes, can't be the string
+    "None", or exceed 64 characters. You can't delete environments, but you
+    can [hide](/product/sentry-basics/environments/#hidden-environments) them.
+
+    // Succinct explanation with key technical details and links into the product content.
+
+    <PlatformContent includePath="set-environment" />
+
+    // An include with a code sample specific to the platform or SDK
+       being viewed by the user.
+
+    Environments help you better filter issues, releases, and user feedback in
+    the Issue Details page of sentry.io, which you learn more about in our
+    [documentation that covers using environments](/product/sentry-basics/environments/).
+
+    // Succinct explanation of the value of environments, with links to more information.
+
+### A Second Example, Avoiding Duplication
+
+The common Getting Started content examplifies further how we avoid duplication in our content with extensive tuning. This file is `/platforms/common/index.md`:
```

Comment:
```suggestion
The common Getting Started content exemplifies further how we avoid duplication in our content with extensive tuning. This file is [`src/platforms/common/index.mdx`](https://github.com/getsentry/sentry-docs/blob/master/src/platforms/common/index.mdx):
```

Are there two possible spellings? I'm more familiar with `exemplifies`.

Reply:
No, that's a typo. Thank you for finding it.